QueryBuilder.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+namespace CanalTP\NavitiaPhp;
+
+class QueryBuilder
+{
+    /**
+     * @var string
+     */
+    private $coverage = null;
+
+    /**
+     * @var string
+     */
+    private $api = null;
+
+    /**
+     * @param string $coverage
+     *
+     * @return self
+     */
+    public function coverage($coverage)
```

Comment:
I know and I think it's a bit strange.
I rather like to have : $queryBuilder->setCoverage('fr-idf')->addApi('')->...